src/pkg/packaging/deb/package.targets

```diff
@@ -4,12 +4,19 @@
 
   <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll"/>
 
+  <PropertyGroup>
+    <IsDebianBasedDistro Condition="$(PackageTargetRid.StartsWith('ubuntu')) or
```

Comment:
It is set really high up in the repo:

https://github.com/dotnet/core-setup/blob/master/src/dir.props#L20-L30

100% agree this is a mess.  If there is something better that I can do now, let me know.